lib/sensors/src/lib.rs

```diff
@@ -2,3 +2,13 @@
 
 pub mod keyence;
 pub mod temperature;
+
+#[derive(PartialEq, Debug)]
```

Comment:
maybe `#[must_use]`?

Reply:
hadn't heard of this before - nice! will add

Reply:
i believe it's used for `Result`s and a few other things in the stdlib